land-queue/land_queue/views.py

```diff
@@ -0,0 +1,46 @@
+import json
+from flask import request, Blueprint, redirect
+from land_queue.handlers.pull_request_handler import PullRequestHandler
+from land_queue.handlers.installation_handler import InstallationHandler
+from land_queue.handlers.integration_handler import IntegrationHandler
+from land_queue.config import Config
+
+
+views = Blueprint("webhook_views", __name__)
+
+
+WEBHOOK_EVENT_HANDLERS = {
+    "installation": InstallationHandler(),
+    "integration_installation": InstallationHandler(),
+    "pull_request": PullRequestHandler(Config),
+}
+
+
+@views.route("/webhooks", methods=["POST"])
+def webhooks():
+    github_event = request.headers.get("X-GitHub-Event")
+
+    if github_event in WEBHOOK_EVENT_HANDLERS:
+        handler = WEBHOOK_EVENT_HANDLERS[github_event]
+        handler.process(request.data)
```

Comment:
Instead of `request.data` you can have `request.json` - this allows you to omit additional lines in handlers as the data will be already loaded from JSON ;)
```suggestion
        handler.process(request.json)
```